src/mol-plugin-state/builder/structure/hierarchy-preset.ts

```diff
@@ -112,7 +114,8 @@ const allModels = TrajectoryHierarchyPresetProvider({
             structures.push(structure);
 
             const quality = structure.obj ? getStructureQuality(structure.obj.data, { elementCountFactor: tr.frameCount }) : 'medium';
-            await builder.representation.applyPreset(structureProperties, params.representationPreset || 'auto', { theme: { globalName: 'model-index' }, quality });
+            const representationPreset = params.representationPreset || plugin.config.get(PluginConfig.Structure.DefaultRepresentationPreset) || PresetStructureRepresentations.auto.id;
```

Comment:
`` || PresetStructureRepresentations.auto.id`` is redundant because ``PluginConfig.Structure.DefaultRepresentationPreset`` will return the default value (``auto``) if not set.

Reply:
Right, but this is not type checked at the moment as `PluginConfigItem<T>.defaultValue` can be `undefined`. Should we change that and revisit all the usage of `plugin.config.get`?

Reply:
Can just add ``!`` since you know that the config variable has a default value set. Tho the extra || is fine too I guess.